### ISSUE TITLE
Getting avrdude: stk500_getsync(): not in sync: resp=0x00 while using FTDI on a razor board without firmware

### ISSUE THREAD

**Issue by DigitalGabriele** [ptrbrtz/razor-9dof-ahrs#48]
Hello, 

Just as the title claims, I have followed all the instructions and when I tried to upload the verified code to razor board, it would keep on complaining about "avrdude: stk500_getsync(): not in sync: resp=0x00". Furthermore, it does not have any firmware on it. There is a chance that it may have been burnt but I would like to rule out other issues too. 

Has anyone ever faced this issue?

**Comment by maharshi114** [comment]
Yes! After using approximately four-month, suddenly the serial data not coming on Serial port. I upload the default code again and showing me the same problem.  

Also, I upload Arduino Blink code but showing me the same problem. 

@DigitalGabriele your problem is solved? 

Thank you.

**Comment by SoorajR-ai** [comment]
Does this problem solved?

**Comment by DigitalGabriele** [comment]
No, I haven't found a solution. I still keep getting the same problem.
